EIPS/eip-5023.md

```diff
@@ -0,0 +1,145 @@
+---
+eip: 5023
+title: Shareable non-transferable non-fungible token standard
+description: A standard interface for shareable non-transferable NFTs
+author: ATARCA team
```

Comment:
The format of author  should  be  changed.  
reference:  https://github.com/ethereum/EIPs/blob/master/eip-template.md